test/local/db.js

```diff
@@ -123,34 +122,23 @@ describe('db with redis disabled', () => {
       post: sinon.spy(() => P.resolve()),
       del: sinon.spy(() => P.resolve())
     }
-
-    redis = {
-      on: sinon.spy(),
-      getAsync: sinon.spy(() => P.resolve(results.redis)),
-      setAsync: sinon.spy(() => P.resolve()),
-      del: sinon.spy(() => P.resolve())
-    }
-
     log = mocks.mockLog()
     tokens = require(`${LIB_DIR}/tokens`)(log, { tokenLifetimes })
     const DB = proxyquire(`${LIB_DIR}/db`, {
       './pool': function () { return pool },
-      redis: { createClient: () => redis }
-    })({ tokenLifetimes, redis: {enabled: false} }, log, tokens, {})
+      './redis': () => {}
+    })({ tokenLifetimes }, log, tokens, {})
     return DB.connect({})
-      .then(result => {
-        assert.equal(redis.on.callCount, 0, 'redis.on was not called')
-
-        db = result
-      })
+      .then(result => db = result)
   })
 
   it('should not call redis when reading sessions', () => {
     results.pool = []
     return db.sessions('fakeUid')
       .then(result => {
-        assert.equal(pool.get.lastCall.args[0], '/account/fakeUid/sessions')
-        assert.equal(redis.getAsync.lastCall, null)
+        assert.equal(pool.get.callCount, 1)
+        assert.equal(pool.get.args[0].length, 1)
+        assert.equal(pool.get.args[0][0], '/account/fakeUid/sessions')
```

Comment:
Yep, it's implied because they don't have a redis object to invoke methods on. I'll come up with better test titles to make that relationship clearer.